src/bundle-generator.ts

```diff
@@ -440,22 +436,33 @@ function updateResultForRootSourceFile(params: UpdateParams, result: CollectingR
 
 		if (isExportAssignment || isReExportFromImportable) {
 			result.statements.push(statement);
+
+			continue;
 		}
 
 		// export { foo, bar, baz as fooBar }
 		if (ts.isExportDeclaration(statement) && statement.exportClause !== undefined && ts.isNamedExports(statement.exportClause)) {
 			for (const exportItem of statement.exportClause.elements) {
+				// export { default }
 				if (exportItem.name.getText() === 'default' && exportItem.propertyName === undefined) {
-					// export { default }
-					// return export { /get name of exportItem.name's symbol node/ as default };
-					// it seems unnecessary?
+					// Leave `export { default } from 'external-package'` untouched
+					const exportedNode = params.resolveIdentifier(exportItem.name);
+					if (!exportedNode || isDeclarationFromExternalModule(exportedNode.getSourceFile())) {
```

Comment:
```suggestion
					const exportedNode = params.resolveIdentifier(exportItem.name);
					if (exportedNode === undefined) {
```

from my understanding

> Leave `export { default } from 'external-package'` untouched

this is already handled by added a `continue` keyword above. do you have a case when this condition works as well?